src/main/resources/application.properties

```diff
@@ -25,7 +25,7 @@ datarepo.resourceId=broad-jade-dev
 datarepo.userEmail=jhert.dev@gmail.com
 datarepo.dnsname=some.dnsname.org
 datarepo.userId="999999999999"
-datarepo.maxStairwayThreads=10
+datarepo.maxStairwayThreads=20
```

Comment:
Are there any drawbacks to increasing it even more?

Reply:
We don't know. This is an area where we need some perf work. How much memory does a thread consume? If it varies a lot by type of work (load, create snap, drs lookup), then do we need different thread pools with different sizes?

Given what I saw on my laptop, we could run a whole lot of load threads at the same time no sweat.